prediction_market_agent/agents/omen_cleaner_agent/deploy.py

```diff
@@ -0,0 +1,51 @@
+from datetime import timedelta
+
+from prediction_market_agent_tooling.deploy.agent import DeployableAgent
+from prediction_market_agent_tooling.gtypes import HexAddress, HexStr
+from prediction_market_agent_tooling.loggers import logger
+from prediction_market_agent_tooling.markets.markets import MarketType
+from prediction_market_agent_tooling.markets.omen.omen_resolving import resolve_markets
+from prediction_market_agent_tooling.markets.omen.omen_subgraph_handler import (
+    OmenSubgraphHandler,
+)
+from prediction_market_agent_tooling.tools.langfuse_ import observe
+from prediction_market_agent_tooling.tools.utils import utcnow
+
+from prediction_market_agent.utils import APIKeys
+
+CLEANER_TAG = "cleaner"
+
+# List of markets we know aren't resolvable and we fine to ignore them.
+IGNORED_FOR_RESOLUTION: list[HexAddress] = [
+    # Throws `Reverted condition not prepared or found`.
+    HexAddress(HexStr("0xf9b90903e98e68a1e69083490ee14c05444a57b4")),
+]
```

Comment:
Can we resolve this as `INVALID`?

Reply:
It's answered, it just doesn't work to resolve it https://presagio.pages.dev/markets?id=0xf9b90903e98e68a1e69083490ee14c05444a57b4